bindings/python/pymongocrypt/mongocrypt.py

```diff
@@ -230,9 +235,22 @@ def __init(self):
                 self.__crypt, aes_256_ctr_encrypt, aes_256_ctr_decrypt, ffi.NULL):
             self.__raise_from_status()
 
+        if self.__opts.csfle_path is not None:
+            lib.mongocrypt_setopt_set_csfle_lib_path_override(self.__crypt,
+                                                              self.__opts.csfle_path.encode(
+                                                                  "ascii"))
```

Comment:
I think this needs to be `utf8` to handle paths with Unicode characters.

Reply:
Hmm. That is something I did not think about. @ShaneHarvey thoughts? Should I switch back to using ffi.cast along with encode("utf-8")?

Reply:
Yes, this needs to be utf-8. I don't see why a ff.cast would be needed though since it happens automatically.